apps/console/src/features/groups/components/wizard/group-assign-users.tsx

```diff
@@ -439,205 +435,211 @@ export const AddGroupUsers: FunctionComponent<AddGroupUserProps> = (props: AddGr
-                                                    listItem={ user.userName }
-                                                    listItemId={ user.id }
-                                                    listItemIndex={ index }
-                                                    isItemChecked={ checkedAssignedListItems?.includes(user) }
-                                                    showSecondaryActions={ false }
-                                                    data-testid={ `${ testId }-update-selected-users` }
+                                                    image={ getEmptyPlaceholderIllustrations().emptyList }
+                                                    imageSize="tiny"
                                                 />
-                                            );
-                                        })
-                                    }
-                                </TransferList>
-                            </TransferComponent>
-                        </Grid.Row>
-                        { isEdit &&
-                        (<Grid.Row columns={ 1 }>
-                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 8 }>
-                                <Button
-                                    data-testid={ `${ testId }-update-user-list-button` }
-                                    primary
-                                    type="submit"
-                                    size="small"
-                                    className="form-button"
-                                >
-                                    { t("common.update") }
-                                </Button>
+                                            </EmphasizedSegment>
+                                        )
+                                }
                             </Grid.Column>
-                        </Grid.Row>)
-                        }
+                        </Grid.Row>
+                        { addNewUserModal() }
                     </Grid>
-                </Forms>)
+                ) : (
+                    <Forms
+                        onSubmit={ () => {
+                            onSubmit(tempUserList);
+                        } }
+                        submitState={ triggerSubmit }
+                    >
+                        <Grid>
+                            <Grid.Row columns={ 2 }>
+                                <TransferComponent
+                                    data-testid={ `${ testId }-update-user-list-transfer` }
+                                    searchPlaceholder={ t("console:manage.features.roles.addRoleWizard.users." +
+                                        "assignUserModal.list.searchPlaceholder") }
+                                    addItems={ addUser }
+                                    removeItems={ removeUser }
+                                    // TODO: Add two methods to handle the search of each list.
```

Comment:
Is this TODO tracked with another git issue?

Reply:
Not sure about this. This chunk of change is a code formatting fix.